Tests/ApolloTests/TestCustomRequestCreator.swift

```diff
@@ -27,40 +27,4 @@ struct TestCustomRequestCreator: RequestCreator {
 
     return body
   }
-
-  public func requestMultipartFormData<Operation: GraphQLOperation>(for operation: Operation,
```

Comment:
This is no longer a requirement of the `RequestBodyCreator` protocol, so it's been removed